### DISCUSSION TITLE
Changing the enemy faction

### DISCUSSION THREAD

**Discussion by Jaz0314** [Vdauphin/HeartsAndMinds#1589] [Question and answer] [answered]
Hi, 
I'm very new to this and I'm struggling to understand how I can change the enemy faction that is spawning. Any help at all would be greatly appreciated.

**Comment by Vdauphin** [comment]
Hello,

You can check in mission parameter all factions available
Then note the number next to this faction and change the default number by the one you choose: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L150 

Cheers

**Reply by Jaz0314** [reply]
Thanks for the answer. My only other question is adding a second faction as simple as copying that chunk of code and pasting it right under with the second factions number put in?

**Reply by Vdauphin** [reply]
Not sure what you want , if you want to add new faction take a look here: http://vdauphin.github.io/HeartsAndMinds/Add-factions

**Reply by Jaz0314** [reply]
Basically what I’m asking is if I could just change semicolon to a comma and add another faction I.D. And then put a semicolon after that. 
Would that make a mixture both factions spawn or do I have to follow the instructions in the linked you posted to do that. 
https://github.com/Vdauphin/HeartsAndMinds/blob/b129b93dba3ed20b8e05a773c8b5fec506f47028/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L150
   For example
   Line.  150                  default = 11,13;

**Reply by Vdauphin** [reply, marked as answer]
You can do it here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L632
Just read the comment

**Reply by Jaz0314** [reply]
Thanks so much. My group is gonna love this!
